packages/server/api/routes/teamMembers.router.js

```diff
@@ -0,0 +1,78 @@
+/* eslint-disable */
+const express = require('express');
+
+const router = express.Router();
+
+const teamMembersController = require('../controllers/teamMembers.controller');
+
+const teamMembersRouter = express.Router();
+
+// Get all team members
+router.get('/', async (req, res, next) => {
+  try {
+    const teamMembers = await teamMembersController.getAllTeamMembers();
+    res.json(teamMembers);
+  } catch (error) {
+    next(error);
+  }
+});
+
+// Get team member by ID
+router.get('/:id', async (req, res, next) => {
+  const memberId = req.params.id;
```

Comment:
Same here with the access rights 😄

Reply:
Also just an FYI, if we return something else than a `404` for an anauthorized user they could guess the user IDs.
example:
 I don't have access and start calling this endpoint, with random numbers. Sometimes it would return a `404` sometimes it would return a `403` maybe, that means I correctly guessed a user ID

Reply:
You don't need to implement theres, this is just some extra info :)